client/post-editor/editor-sidebar/feedback-view.jsx

```diff
@@ -0,0 +1,84 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes, PureComponent } from 'react';
+import { localize } from 'i18n-calypso';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import FeedbackSidebarHeader from './feedback-header';
+import FeedbackRequestForm from './feedback-request-form';
+import FeedbackList from './feedback-list';
+import SidebarFooter from 'layout/sidebar/footer';
+
+// TODO: Find a clearer word than "share" for the owner of feedback and use it in subcomponent and CSS classes
+class FeedbackView extends PureComponent {
+	static propTypes = {
+		translate: PropTypes.func,
+		close: PropTypes.func,
+		sharedLinks: PropTypes.array
+	}
+
+	constructor() {
+		super();
+		this.state = { openFeedbackCount: 0 };
+	}
```

Comment:
I saw you'd enabled the class properties transform but hadn't thought of using it here. It reads more clearly to me, and I made the change.